easybuild/easyconfigs/g/GROMACS/GROMACS-2018.3-CrayGNU-19.10-PLUMED-2.4.3-cuda-10.1.eb

```diff
@@ -0,0 +1,44 @@
+# contributed by Luca Marsella (CSCS)
+#
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2013 University of Luxembourg / LCSB, Cyprus Institute / CaSToRC, Ghent University
+# Authors::   Wiktor Jurkowski <wiktor.jurkowski@uni.lu>, Fotis Georgatos <fotis.georgatos@uni.lu>, \
+#             George Tsouloupas <g.tsouloupas@cyi.ac.cy>, Kenneth Hoste
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-93.html
+#
+name = 'GROMACS'
+version = '2018.3'
+versionsuffix = '-PLUMED-2.4.3-cuda-10.1'
+
+homepage = 'http://www.gromacs.org'
+description = """GROMACS is a versatile package to perform molecular dynamics,
+ i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles."""
+
+toolchain = {'name': 'CrayGNU', 'version': '19.10'}
+toolchainopts = {'opt': True, 'usempi': True, 'pic': True, 'verbose': False, 'openmp': True}
+
+source_urls = ['ftp://ftp.%(namelower)s.org/pub/%(namelower)s/']
+sources = [SOURCELOWER_TAR_GZ]
+patches = ['%(namelower)s-%(version_major)s.2.patch']
+
+# CMake and libxml2 dependencies with dummy toolchain
+builddependencies = [
+    ('CMake', '3.14.5', '', True),
```

Comment:
If you are using this CMake, instead of the 3.12, one. Please check the easybuild log. You need to patch GROMACS if it is not automatically detecting the system. (https://redmine.gromacs.org/issues/2927)

Reply:
The message `-- Did not detect build CPU vendor - detection program did not compile` (mentioned in https://redmine.gromacs.org/issues/2927) can be found in the log, e. g. line 386: /scratch/snx3000/jenscscs/TestingEB/daint-gpu/2b057e8-2901/software/GROMACS/2018.3-CrayGNU-19.10-PLUMED-2.4.3-cuda-10.1/easybuild/easybuild-GROMACS-2018.3-20191111.143456.log.

Reverting to CMake 3.12.0

Reply:
Thanks @victorusu for pointing that out.

@kraushm This problem can be easily overlooked in a future update, so would you mind adding a comment in the recipe in order to keep track of the issue?

Something like: "other CMake versions are known to fail, see https://github.com/eth-cscs/production/pull/1411"